suitcase-{{ cookiecutter.subproject_name }}/.travis.yml

```diff
@@ -13,8 +13,9 @@ install:
   - pip install -r requirements-dev.txt
 
 script:
-  - coverage run -m pytest  # Run the tests and check for test coverage.
-  - coverage report -m  # Generate test coverage report.
+  - coverage run -m pytest suitcase/{{ cookiecutter.subpackage_name }}/tests.py
```

Comment:
Revisiting https://github.com/NSLS-II/suitcase-cookiecutter/issues/4 it would be better to configure this via a configuration file so that users can run plain `pytest` locally and have it find the right files. See https://github.com/NSLS-II/suitcase-tiff/pull/5/commits/b2c42b40db857b03edb524bc20245c8eea125081.